data_and_models/pipelines/relation_extraction/convert_chemprot_fmt.py

```diff
@@ -251,7 +288,17 @@ def annotate_scibert(row):
         entity_2 = text[start_2:end_2]
         part_3 = text[end_2:]
 
-        text = part_1 + "<< " + entity_1 + " >>" + part_2 + "[[ " + entity_2 + " ]]" + part_3
+        text = (
+            part_1
+            + "<< "
+            + entity_1
+            + " >>"
+            + part_2
+            + "[[ "
+            + entity_2
+            + " ]]"
+            + part_3
+        )
```

Comment:
Maybe this one can be made into an f-string?

Reply:
Done in 99b63f4.